RELEASE-NOTES.md

```diff
@@ -24,6 +24,61 @@ replication. The rest of the changes are fixes (riak-core claim,
 repl), new features (gsets, participate in coverage, node-confirms),
 and fixes to tests and the build/development process.
 
+#### Multi Datacentre Replication
+
+Previously a paid for enterprise addition to Riak, as part of RiakEE,
```

Comment:
Riak EE

Reply:
Well, *they* at basho rebranded it RiakEE, RiakKV, RIakCS, but I'll change it back, gladly

Reply:
Ha ha, yeah Basho marketing. I tried to explain the various branding to a colleague from the marketing department yesterday afternoon. Her, "so, they renamed Riak CS (cloud storage) to Riak S2, because it supported the Amazon S3 API, but wasn't quite as good?". Me, "I can't think of any other explanation". 😆

Reply:
Hahaha